packages/web/src/components/GraphQLHooksProvider.test.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment jsdom
+ */
+
```

Comment:
I would love a walk through of this in the core meeting.

Reply:
Sure thing. Do remind me as well. 

The gist is this:
- Jest now defaults to node environment. Need to specify otherwise.
- For most Redwood packages that require jsdom, like router, I handled this setting in jest.cofig.
- We only have two test files in Web and no jest.config. Used this docblock method for convenience + example.

Note:
- I set the project Web jest.config to be jsdom